### PR TITLE
Missing declared

### PR DESCRIPTION
**Pull request by ariel11**
**Type:** Missing

**Summary:**
Missing declared

**Details:**
Followed source link.  "Copyright" file is MIT.

**Resolution:**
Declared MIT

**Affected definitions**:
- [libxml2 4c2e7c651f6c2f0d1a74f350cbda95f7df3e7017](https://clearlydefined.io/definitions/git/github/gnome/libxml2/4c2e7c651f6c2f0d1a74f350cbda95f7df3e7017/4c2e7c651f6c2f0d1a74f350cbda95f7df3e7017)

### CODE REVIEW

**Comment by AE49** [comment]
@ariel11 should this be "Other" for Declared License because of the extra text added to the MIT license?

File: curations/git/github/gnome/libxml2.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+coordinates:
+  name: libxml2
+  namespace: gnome
+  provider: github
+  type: git
+revisions:
+  4c2e7c651f6c2f0d1a74f350cbda95f7df3e7017:
+    licensed:
+      declared: MIT
```
Comment (pombredanne) [inline curations/git/github/gnome/libxml2.yaml]: @ariel11 I think the declared license in the `Copyright` is `X11` not `MIT` 
See https://raw.githubusercontent.com/GNOME/libxml2/4c2e7c651f6c2f0d1a74f350cbda95f7df3e7017/Copyright

This is detected exactly as such by ScanCode at least.

**Comment by pombredanne** [comment]
@AE49 

> should this be "Other" for Declared License because of the extra text added to the MIT license?

In this case, I think there is no such need as this does not introduces any new terms:

https://github.com/GNOME/libxml2/blob/4c2e7c651f6c2f0d1a74f350cbda95f7df3e7017/Copyright

> Except where otherwise noted in the source code (e.g. the files hash.c,
list.c and the trio files, which are covered by a similar licence but
with different Copyright notices) all the files are:


This is more an intro text saying more or less `this is the declared license...`

File: curations/git/github/gnome/libxml2.yaml (hunk 1 above)
Comment (pombredanne) [reply curations/git/github/gnome/libxml2.yaml]: Thanks!

Comment (ariel11) [reply curations/git/github/gnome/libxml2.yaml]: Yes, I agree.  I updated the .yaml to X11.

**Pull request merged**